src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -339,52 +355,50 @@ uct_cuda_ipc_md_open(uct_component_t *component, const char *md_name,
         .detect_memory_type = ucs_empty_function_return_unsupported
     };
 
-    uct_cuda_ipc_md_t* md;
-    uct_cuda_ipc_component_t* com;
-
-    md = ucs_calloc(1, sizeof(uct_cuda_ipc_md_t), "uct_cuda_ipc_md");
+    uct_md_t *md = ucs_calloc(1, sizeof(*md), "uct_cuda_ipc_md");
```

Comment:
minor: typically we do not allocate mem as initialization, i'd just declare a var here

Reply:
ok